components/list/list-item-mixin.js

```diff
@@ -281,10 +275,6 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 				width: 100%;
 				z-index: 5;
 			}
-			:host([draggable][selected]:not([disabled])) .d2l-list-item-active-border,
-			:host([draggable][selected]:not([disabled])) d2l-list-item-generic-layout.d2l-focusing + .d2l-list-item-active-border {
-				transform: rotate(1deg);
-			}
```

Comment:
Would this enable us to remove the surrounding `<div class="d2l-list-item-drag-image">` if drag & drop isn't enabled? I had that question the last time I was in list and I think I came to the conclusion that we couldn't but it was because of this CSS.

Since list items get duplicated SO many times in large lists, the fewer DOM nodes -- especially for simple lists -- the better.

Reply:
Good question.  That element currently also wraps the "<div class="d2l-list-item-active-border"></div>" element which would currently get included in the drag image.  I can play around with removing it.

Reply:
So, the answer is, not without affecting the color of the bottom border in the drag image when a selected item is dragged.  The color difference is very subtle with the transparency but it is different, and if we ever changed the color it might be more noticeable.

Reply:
> ...not without affecting the color of the bottom border in the drag image when a selected item is dragged

Sorry, I meant keep the `<div>` but only render it when `draggable` is set.

Reply:
Yeah we should be able to do that, though I think Lit will generate two comment nodes to mark the part that results from conditional rendering.

Reply:
Even if you do something like:

```javascript
const innerView = html`<inner list view>`;
return this.draggable ? html`<div class="d2l-list-item-active-border">${innerView}</div>` : innerView;
```